x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/hooks/setup_technology.ts

```diff
@@ -108,7 +110,8 @@ export function useSetupTechnology({
   );
 
   useEffect(() => {
-    if (isEditPage) {
+    if (isEditPage && agentPolicies && agentPolicies.some((policy) => policy.supports_agentless)) {
+      setSelectedSetupTechnology(SetupTechnology.AGENTLESS);
```

Comment:
Everything looks good! I have one question when we add agent-based and agentless integrations. When we go to edit the Agent-based integration, will the setupTechnology be set to `Agentless` or `Agent based`?